src/vmm/src/acpi/mod.rs

```diff
@@ -263,7 +263,7 @@ pub mod tests {
     fn test_write_acpi_table_small_memory() {
         let mut vmm = default_vmm();
         vmm.guest_memory = arch_mem(
-            (SYSTEM_MEM_START + SYSTEM_MEM_SIZE - 4096)
+            (SYSTEM_MEM_START + SYSTEM_MEM_SIZE - 8192)
```

Comment:
General comment to the whole commit: to these changes fix smith broken in previous ones? In that case would be nice to move necessary changes to commits which need them.

Reply:
Yes, ACPI tables are larger because of the extra CPUs and the hotplug device, and MMIO addresses have changed because the CPU container is assigned an MMIO address first. I'll move them to their respective commits